doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1393,11 +1393,11 @@ Chapter :ref:`Tactics`. The basic assertion command is:
       The name you provided is already defined. You have then to choose
       another name.
 
-   .. exn:: Nested proofs are not allowed unless you turn option Nested Proofs Allowed on.
+   .. exn:: Nested proofs are not allowed unless you turn the :flag:`Nested Proofs Allowed` flag on.
```

Comment:
Sorry to insist: please undo the change on this line, or fix the error message in the code as well. Documentation of error messages must reflect exactly their content, even if the contents are flawed.